src/helpers.rs

```diff
@@ -54,6 +61,43 @@ pub fn copy_dir(source: &Path, dest: &Path) -> Result<()> {
     Ok(())
 }
 
+/// Find the root zine file in current dir
+fn zine_file_existed(path: &PathBuf) -> bool {
+    for entry in WalkDir::new(path).max_depth(1) {
+        let entry = entry.unwrap();
+        if entry.path().is_file() {
+            let root_file = entry.file_name().to_str().unwrap();
+            // Insure the name
+            if !root_file.eq(crate::ZINE_FILE) {
```

Comment:
I think `entry.file_name().filter(|name| *name == create::ZINE_FILE).is_some()` is more efficiency